distribution/docker/src/docker/Dockerfile

```diff
@@ -40,11 +40,13 @@ RUN chmod 0660 config/elasticsearch.yml config/log4j2.properties
 #
 # The tini GitHub page gives instructions for verifying the binary using
 # gpg, but the keyservers are slow to return the key and this can fail the
-# build. Instead, we check the binary against a checksum that we have
-# computed.
-ADD https://github.com/krallin/tini/releases/download/v0.18.0/tini${tini_suffix} /tini
-COPY config/tini${tini_suffix}.sha512 /tini.sha512
-RUN sha512sum -c /tini.sha512 && chmod +x /tini
+# build. Instead, we check the binary against a checksum that they provide.
+RUN wget --no-check-certificate --no-cookies --quiet https://github.com/krallin/tini/releases/download/v0.19.0/tini-\$(dpkg --print-architecture) \
+    && wget --no-check-certificate --no-cookies --quiet https://github.com/krallin/tini/releases/download/v0.19.0/tini-\$(dpkg --print-architecture).sha256sum \
+    && echo "\$(cat tini-\$(dpkg --print-architecture).sha256sum)" | sha256sum -c \
+    && rm -f tini-\$(dpkg --print-architecture).sha256sum \
```

Comment:
There's no need to remove the checksum, since this is just the builder image.

Reply:
The reason I remove it was were still under the working dir of `/usr/share/elasticsearch`, which gets copied into the main image. Thus the checksum will be copied to the second stage.

What would be easier is to move the whole Tini "block" of code up before you change the `WORKDIR` to `/usr/share/elasticsearch`, around line 20. Do you mind if I move the Tini code "block" to line 20? I did not want to move to much stuff around, but if you think is a okay solution I will!

Reply:
In my latest commit, I moved the Tini code "block" up in the Dockerfile before you change `WORKDIR`, which eliminates the need to delete the checksum.

Reply:
I like that we removed the checksum file even if it's currently unneeded, because it means if there's a later refactoring that would for example, accidentally revert executing this in `/usr/share/elasticsearch`, then we don't have to "remember" to now delete the checksum file. It can be done as part of this `RUN` command, so not introducing a new layer (which doesn't even matter since it's in the builder image).

Reply:
@jasontedor @pugnascotia I am flexiable either way. 

> It can be done as part of this RUN command, so not introducing a new layer (which doesn't even matter since it's in the builder image).

I think this is key. If its no new layer were not adding to the build time or build size. Plus, like @jasontedor, its just the builder image.